assets/mixs-patterns-materialized.yaml

```diff
@@ -13970,7 +13970,7 @@ slots:
     description: Previous land use and dates
     title: history/previous land use
     examples:
-    - value: fallow; 2018-05-11:T14:30Z
+    - value: follow; 2018-05-11:T14:30Z
```

Comment:
this is an incorrect conversion the word is meant to be fallow (meaning not sown with seed) not follow.